src/nix/install.sh

```diff
@@ -61,10 +55,6 @@ else
         find_prev_version_from_git_tags VERSION https://github.com/NixOS/nix "tags/"
         curl -sSLf -o "${tmpdir}/install-nix" https://releases.nixos.org/nix/nix-${VERSION}/install
     fi
-    curl -sSLf -o "${tmpdir}/install-nix.asc" https://releases.nixos.org/nix/nix-${VERSION}/install.asc
-    cd "${tmpdir}"
-    receive_gpg_keys NIX_GPG_KEYS
```

Comment:
Can we remove this function? 

https://github.com/devcontainers/features/blob/3921f5917148583af38d1d96a1b7e0f31d22181c/src/nix/utils.sh#L86

Reply:
resolved in aa00f714313489f3d3eb2c3c738870023cf6302d